content/correspondence/reference/API endpoints/mark as read/_index.en.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Mark as read
+linktitle: Mark as read
+description: Endpoint for giving the correspondence status "Read".
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence/{{correspondenceId}}/markasread
+
+## Description
+
+This endpoint sets the status of a existing correspondence as "Read". Before using this endpoint, the status of the correspondence must be "Fetched". The status of a correspondence can be checked by using the details endpoint described [here](https://docs.altinn.studio/api/correspondence/spec/#/Correspondence/get_correspondence_api_v1_correspondence__correspondenceId__details).
+
```

Comment:
Should reference overview, not details. 

Details should mostly be used to finding errors or when extra info is needed.